docs/theoryguide/understanding_ray_history.rst

```diff
@@ -0,0 +1,58 @@
+Understanding ray history state in DAG-MCNP5/6
+==============================================
+
+Note that all the information here applies to both DAG-MCNP5 and DAG-MCNP6.
+
```

Comment:
```suggestion

DAGMC uses a ray history to improve robustness against situations when floating point arithmetic and logical comparisons can result in inconsistent or ambiguous outcomes.  The ray history is a list of facets that were crosses since the last collision/change in direction of the particle.
```